client/components/place/place-list/place-list.component.js

```diff
@@ -107,6 +111,7 @@ angular.module('placeList', ['filterMapType', 'popularTracks', 'ngAnimate'])
         ctrl.hidePopularTracks = true;
 
         ctrl.checkPopularPlaces = function() {
+          angular.element('.popular-places-wrapper').css('display', 'block');
```

Comment:
There is already ng-hide :) It's not work correctly. That's I decided to insert display: none, as you wished yesterday week. We spoke about it with you